ducc/unitfiles/cvmfs_ducc.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=DUCC Daemon Unpacker Containers in CVMFs
+After=network.target dockerd.service
+Requires=dockerd.service
+
+[Service]
+Environment="RECIPE_FILE=YourRecipeFile_UPDATE_THIS_VALUE.yaml"
+Environment="DOCKER2CVMFS_DOCKER_REGISTRY_PASS=ThePassOfTheUserForTheImageRegistry_UPDATE_THIS_VALUE"
```

Comment:
Let's discuss the parameter names, I think it would be good to have a common prefix, either CVMFS or DUCC